adyen/assets/logs/adyen.yaml

```diff
@@ -0,0 +1,83 @@
+id: adyen
+metric_id: adyen
+backend_only: false
+facets:
+  - groups:
+      - Event
+    name: Event Name
+    path: evt.name
+    source: log
+pipeline:
+  type: pipeline
+  name: Adyen
+  enabled: true
+  filter:
+    query: "source:adyen"
+  processors:
+    - type: string-builder-processor
+      name: Map `notificationItems.NotificationRequestItem.eventCode` to `evt.name`
+      enabled: true
+      template: "%{notificationItems.NotificationRequestItem.eventCode}"
+      target: evt.name
+      replaceMissing: false
+    - type: string-builder-processor
+      name: Map `notificationItems.NotificationRequestItem.eventDate` to `eventDate`
```

Comment:
I believe this could lead to having multiple comma-separated values in `eventDate` in case `notificationItems` contains multiple entries, which might cause issues when using the field as the official date for the log later on.

Reply:
Log will contain only one NotificationRequestItem object in the `notificationItems` array as per Adyen webhook doc. It is also mentioned in the integration README to configure the webhook in JSON format.
https://docs.adyen.com/development-resources/webhooks/webhook-types/

![image](https://github.com/user-attachments/assets/41439ba7-2d4d-41c1-94fc-957747c7dce0)

Reply:
Thanks for confirming there is only 1 element and providing the reference link!